### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by renefritze**
This currently fails compiling against deal.II 9.2
Is that expected @DavidSCN?

### CODE REVIEW

File: .github/workflows/indent.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Indent
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    branches: [ master, develop ]
```
Comment (DavidSCN) [inline .github/workflows/indent.yml:7]: For some reason I named the default branch now `main` and not `master`

Comment (renefritze) [reply .github/workflows/indent.yml:7]: New default in git. The other action should run though.

Comment (DavidSCN) [reply .github/workflows/indent.yml:7]: Wait, I guess I need to enable the GH CI explicitly.

Comment (DavidSCN) [reply .github/workflows/indent.yml:7]: ..maybe you need more permission to do so. I cannot trigger the CI here manually. Just invited you as collaborator, which might help.

**Pull request merged**